packages/subgraph/src/utils/erc165.ts

```diff
@@ -3,7 +3,8 @@ import {ethereum, Bytes} from '@graphprotocol/graph-ts';
 export function supportsInterface(
   contract: ethereum.SmartContract,
   interfaceId: string,
-  expected = true
+  // eslint-disable-next-line @typescript-eslint/no-inferrable-types
+  expected: boolean = true
```

Comment:
the `boolean` part is not necessary and therefore also not the eslint disable comment

Reply:
agree, but when is compiled to WASM the compiler complains, 
that is why i disabled the eslint for it